plugin.py

```diff
@@ -58,11 +66,50 @@ def platform() -> str:
         return "unknown-linux-gnu"
 
 
+def open_runnables_in_terminus(window: Optional[sublime.Window], runnables: List[Runnable]) -> None:
+    filtered_runnables = [r for r in runnables if r["kind"] == "cargo"]
+    if len(filtered_runnables) == 0:
+        return
+    if not window:
+        return
+    view = window.active_view()
+    if not view:
+        return
+    if not Terminus:
+        sublime.error_message(
+            'Cannot run executable. You need to install the "Terminus" package and then restart Sublime Text')
+        return
+    main_cargo = shutil.which("cargo")
+    if not main_cargo:
+        sublime.error_message('Cannot find "cargo" on path.')
+        return
```

Comment:
As @rwols commented in his PR, we probably don't have to check that (https://github.com/sublimelsp/LSP-rust-analyzer/pull/16#discussion_r652077885)